### ISSUE TITLE
can't build the docs locally

### ISSUE THREAD

**Issue by tpapp** [JuliaStats/DataFrames.jl#897]
Using Julia `Version 0.4.1 (2015-11-08 10:33 UTC)`.

```julia
tamas@tamas ...v0.4/DataFrames/docs % julia build.jl
Docile: updating package list...
WARNING: both StatsBase and Base export "histrange"; uses of it in module DataArrays must be qualified
WARNING: both StatsBase and Base export "midpoints"; uses of it in module DataArrays must be qualified
WARNING: both DataArrays and StatsBase export "autocor"; uses of it in module DataFrames must be qualified
WARNING: both DataArrays and StatsBase export "histrange"; uses of it in module DataFrames must be qualified
WARNING: both DataArrays and StatsBase export "inverse_rle"; uses of it in module DataFrames must be qualified
WARNING: both DataArrays and StatsBase export "midpoints"; uses of it in module DataFrames must be qualified
WARNING: both DataArrays and StatsBase export "rle"; uses of it in module DataFrames must be qualified
WARNING: both Interface and Base export "Docs"; uses of it in module Main must be qualified
ERROR: LoadError: UndefVarError: Docs not defined
 in find_loaded_packages at /home/tamas/.julia/v0.4/Docile/src/Cache/types.jl:44
 in update! at /home/tamas/.julia/v0.4/Docile/src/Cache/types.jl:55
 in update! at /home/tamas/.julia/v0.4/Docile/src/Cache/types.jl:67
 in loadedmodules at /home/tamas/.julia/v0.4/Docile/src/Cache/types.jl:76
 in metadata at /home/tamas/.julia/v0.4/Docile/src/Interface/legacy.jl:39
 in include at ./boot.jl:261
 in include_from_node1 at ./loading.jl:304
 in process_options at ./client.jl:280
 in _start at ./client.jl:378
while loading /home/tamas/.julia/v0.4/DataFrames/docs/build.jl, in expression starting on line 62
```

**Comment by tshort** [comment]
It's likely that we're out of sync with Docile.

**Comment by tshort** [comment]
Fixed by #929.
